Dockerfile

```diff
@@ -33,34 +33,44 @@ FROM python:3.8-slim
 ARG CRYPTOGRAPHY_DONT_BUILD_RUST=1
 
 # Re #93, #73, excluding rustc (adds another 430Mb~)
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    libssl-dev \
-    libffi-dev \
-    gcc \
-    libc-dev \
-    libxslt-dev \
-    zlib1g-dev \
-    g++
+RUN set -ex; \
+    apt-get update && apt-get install -y --no-install-recommends \
+        g++ \
+        gcc \
+        gosu \
+        libc-dev \
+        libffi-dev \
+        libssl-dev \
+        libxslt-dev \
+        zlib1g-dev && \
+    rm -rf /var/lib/apt/lists/*; \
+    useradd -u 911 -U -d /datastore -s /bin/false changedetection && \
```

Comment:
why `911` ? :) Is it better to choose a UID > 1000 ?

Reply:
It's the standard one used by [linuxserver.io](https://github.com/linuxserver) and makes it a system user, but a higher default one like 3000 is fine as well imho. It can be chosen freely anyways (docker-entrypoint.sh:L16-20).